clang/tools/driver/cc1depscan_main.cpp

```
@@ -833,7 +833,10 @@ void ScanServer::start(bool Exclusive, ArrayRef<const char *> CASArgs) {
       return;
     SmallString<64> LLVMCasStorage;
     SmallString<64> CASPath;
-    CASOpts.getResolvedCASPath(CASPath);
+    if (auto Err = CASOpts.getResolvedCASPath(CASPath)) {
+      llvm::consumeError(std::move(Err));
```

This deserves comment about ignoring the error.  Maybe something like

```
// Failure to resolve a cas path for validation is ignored, because any error will be reported when attempting to open the cas.
```